docs/kratos/hooks/01_configure-hooks.mdx

```diff
@@ -373,10 +376,10 @@ To do that, set `response.ignore` to `true` in the webhook config:
 ```yaml
 - hook: web_hook
   config:
-  // highlight-start
+    # highlight-start
     response:
       ignore: true
-  // highlight-end
+    # highlight-end
```

Comment:
I compared this with the current master and there seems to be no difference in how this is rendered. I used `//` for the line highlighting trigger based on the Docusaurus docs: 
https://docusaurus.io/docs/markdown-features/code-blocks#line-highlighting

So what you did here is just using the yaml-appropriate commenting syntax, correct? The magic comment `higlight-start/end` doesn't seem to care about it.

Reply:
`//` is not a valid YAML comment, so code formatting is broken. YAML comments use hashtags `#`: https://www.tutorialspoint.com/yaml/yaml_comments.htm